api/backend.go

```diff
@@ -304,12 +310,12 @@ func (b *StatusBackend) registerHandlers() error {
 			return nil, err
 		}
 
-		hash, err := b.SendTransaction(ctx, txArgs)
-		if err != nil {
-			return nil, err
+		result := b.SendTransaction(txArgs, password)
```

Comment:
you don't need this handler whatsoever
we don't need to expose `eth_sendTransaction` anymore, it is Clojure's responsibility to call the right native method if `eth_sendTransaction` is called.

Reply:
Ok. so I'll remove the params.SendTransactionMethodName handler from here and remove the RPC tests for e2e transactor tests